site/content/en/docs/Reference/gameserverallocation.md

```diff
@@ -141,13 +129,8 @@ spec:
 
 The `spec` field is the actual `GameServerAllocation` specification, and it is composed as follows:
 
-- `required` is a [GameServerSelector][gameserverselector]
-  (matchLabels. matchExpressions, gameServerState and player filters) from which to choose GameServers from.
-- `preferred` is an ordered list of preferred
-  [GameServerSelector][gameserverselector]
-  that are _optional_ to be fulfilled, but will be searched before the `required` selector.
+- `selectors` is an ordered list of [GameServerSelector][gameserverselector].
```

Comment:
Instead of replacing the old docs immediately, I think it might make more sense to leave the old fields documented but marked as deprecated. We should wait a few releases and then remove the old fields from the docs.

Reply:
Done.